stdlib/influxdata/influxdb/monitor/monitor.flux

```diff
@@ -89,48 +154,91 @@ stateChangesOnly = (tables=<-) => {
         |> experimental.group(mode: "extend", columns: ["_level"])
 }
 
-// StateChanges takes a stream of tables, fromLevel, and toLevel and returns
-// a stream of tables where status has gone from fromLevel to toLevel.
+// stateChanges detects state changes in a stream of data with a _level column and outputs records that change from fromLevel to toLevel.
+//
+// ## Parameters
+// - `fromLevel` is the level to detect a change from. Defaults to "any".
+// - `toLevel` is the level to detect a change to. The function output records that change to this level
+//
+// ## Detect when the state changes to critical
+// ```
+// monitor.from(start: -1h)
+//    |> monitor.stateChanges(toLevel: "crit")
+// ```
 //
-// StateChanges only operates on data with data where r._level exists.
 stateChanges = (fromLevel="any", toLevel="any", tables=<-) => {
     return if fromLevel == "any" and toLevel == "any" then
         tables |> stateChangesOnly()
     else
         tables |> _stateChanges(fromLevel: fromLevel, toLevel: toLevel)
 }
 
-// Notify will call the endpoint and log the results.
-notify = (tables=<-, endpoint, data) => tables
```

Comment:
This is probably what it's complaining about. Looks like we deleted the `notify` function and the `logs` function below. If we add them back in, we should see that test pass again.